### PR TITLE
Code style changes

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

File: example/database/factories/BookFactory.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 use Faker\Generator as Faker;
 
-$factory->define(\ExampleApp\Models\ModelsWithArrayState\Book::class, function (Faker $faker) {
+$factory->define(\ExampleApp\Models\ModelsWithArrayState\Book::class, static function (Faker $faker) {
```
Comment (christophrumpel) [inline example/database/factories/BookFactory.php]: I don't like the static keyword for these functions. Please remove them.

**Comment by christophrumpel** [comment]
Thanks for the PR. Only one thing I don't like, please check the review comment.

**Comment by simonschaufi** [comment]
done

File: example/database/migrations/2020_01_30_0000_create_books_table.php [hunk 2]
```diff
@@ -13,7 +13,7 @@ class CreateBooksTable extends Migration
      */
     public function up()
     {
-        Schema::create('books', function (Blueprint $table) {
+        Schema::create('books', static function (Blueprint $table) {
```
Comment (christophrumpel) [inline example/database/migrations/2020_01_30_0000_create_books_table.php]: There are still lots of changes where you added `static`. Please remove all of them where you added them.

Comment (simonschaufi) [reply example/database/migrations/2020_01_30_0000_create_books_table.php]: Do you use the PhpStorm Plugin https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended- ? This plugin told me that static methods are better. As long as you don't use `$this` in the closure, `static` should be used. 

Sources: 
* https://github.com/slevomat/coding-standard/issues/400
* https://blobfolio.com/2017/09/25-easy-php7-micro-optimizations/ (Use Static Methods and Properties)

Comment (christophrumpel) [reply example/database/migrations/2020_01_30_0000_create_books_table.php]: Yeah I use this package but I don't use the static keyword for closures. I know there might be a small improvement when using, but I don't think it is noticeable and I like my code cleaner and without them.

Comment (simonschaufi) [reply example/database/migrations/2020_01_30_0000_create_books_table.php]: ok fine.

**Comment by christophrumpel** [comment]
Thanks 👍

**Pull request merged**